plugins/source/aws/resources/services/emr/studios.go

```diff
@@ -0,0 +1,57 @@
+package emr
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Studios() *schema.Table {
+	tableName := "aws_emr_studios"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_Studio.html`,
+		Resolver:            fetchEmrStudios,
+		PreResourceResolver: getStudio,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "elasticmapreduce"),
+		Transform:           transformers.TransformWithStruct(&types.Studio{}),
+		Relations:           []*schema.Table{studioSessionMapping()},
+		Columns: []schema.Column{
```

Comment:
I would swap Relations and Columns, also indent relations as multiline so it's easier to spot :)